### ISSUE TITLE
Shape divider curve method does not work

### ISSUE THREAD

**Issue by Agent-11387** [fireship-io/wavy-curvey-blobby-website#6]
I followed your youtube tutorial and the shapedividr method does not work. I am using a chromebook with chrome.

**Comment by surjeet176** [comment]
mine don't work either,  the other contents are hidden in the back of the svg, since it is positioned absolute, how to align the contents properly ?

**Comment by Agent-11387** [comment]
i thin i figured smth out il see my files

**Comment by surjeet176** [comment]
yeah, i figured it out too, i removed all the previous css and played with the raw SVG, it gives us more control over the component flow
